app/views/devise/registrations/new.haml

```diff
@@ -30,12 +32,12 @@
           = image_tag 'icons/exclamation.png', alt: '!', title: ''
           %span.msg
     #signup-secret
-      .sub.vtop= f.label :secret, "Secret (optional)"
+      .sub.vtop= label_tag :secret, "Secret (optional)"
```

Comment:
Why is this changed from a formbuilder label to a label_tag?

Reply:
Ah, because the secret is a text_field_tag?

Reply:
Yeah